sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/FormLineHelper.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.implementation;
+
+import com.azure.ai.formrecognizer.models.Appearance;
+import com.azure.ai.formrecognizer.models.FormLine;
+
+/**
+ * The helper class to set the non-public properties of an {@link FormLine} instance.
+ */
+public final class FormLineHelper {
```

Comment:
what does a non-public property is? I guess I am confused on why the need of this class instead of adding Appearance to the FormLine constructor

Reply:
Same comment for table.boundingbox

Reply:
Yeah, we don't want to introduce public constructors for output only models. Since that could be confusing to the users. So this a new pattern we have introduced wherein we have private setters on the model class and interface implementations to set those properties internally which are never exposed to users.